jsapp/js/keymap.js

```diff
@@ -0,0 +1,6 @@
+export default {
+  APP_SHORTCUTS: {
+    EDGE: 'alt+e',
```

Comment:
What is this for? When I use it I see "Active Projects" and "Deleted (#)" show up, but both of these does nothing when clicked

Reply:
This is a way for us to build features but not expose them to users. It just toggles a class on the body (`is-edge`). For example, under Data, there is a tab for legacy reports. (Some of the things it enables don't do anything though, like the "Deleted (#)", we need to clean that up.)

Reply:
Oh, so `is-edge` means "is bleeding edge"?

Reply:
Yes (I guess it could be confused with IE Edge).